core/src/main/java/org/mule/runtime/core/api/util/CaseInsensitiveHashMap.java

```diff
@@ -65,13 +63,13 @@
     return EMPTY_MAP;
   }
 
-  private final Map<K, V> delegate;
+  protected Map<K, V> delegate;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "delegate" transient or serializable. ([squid:S1948](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1948))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7700&issues=AWkLNLUWotIXnmsUkTg9&open=AWkLNLUWotIXnmsUkTg9)

[ISSUE_KEY]: AWkLNLUWotIXnmsUkTg9